metrics.md

```diff
@@ -0,0 +1,226 @@
+
+If there are any additional metrics that you would like to see, please open an issue in the [GitHub repository](https://github.com/skip-mev/slinky). Note that this document is not fully comprehensive and may be updated in the future. However, it should provide a good starting point for monitoring the side-car.
+
+# Table of Contents
+
+* [Metrics](#metrics)
+    * [Health Metrics](#health-metrics)
+    * [Prices Metrics](#prices-metrics)
+        * [Price Feed Metrics](#price-feed-metrics)
+        * [Aggregated Price Metrics](#aggregated-price-metrics)
+    * [HTTP Metrics](#http-metrics)
+    * [WebSocket Metrics](#websocket-metrics)
+
+# Metrics
+
+> **Definitions**:
+>
+> * **Market**: A market is a pair of assets that are traded against each other. For example, the BTC-USD market is the market where Bitcoin is traded against the US Dollar.
+> * **Price Feed**: A price feed is indexed by a price provider and a market. For example, the Coinbase API provides a price feed for the BTC-USD market.
+> * **Price Provider**: A price provider is a service that provides price data for a given market. For example, the Coinbase API is a price provider for the Coinbase markets.
+> * **Market Map Provider**: A market map provider is a service that supplies the markets that the side-car needs to fetch data for.
+
+The side-car exposes metrics on the `/metrics` endpoint. These metrics are in the Prometheus format and can be scraped by Prometheus or any other monitoring system that supports Prometheus format.
+
+## Health Metrics
+
+There are three primary health metrics that are exposed by the side-car:
+
+* [`side_car_health_check_system_updates_total`](#side_car_health_check_system_updates_total): This metric is a counter that increments every time the side-car updates its internal state. This is a good indicator of the side-car's overall health.
+* [`side_car_health_check_ticker_updates_total`](#side_car_health_check_ticker_updates_total): This metric is a counter that increments every time the side-car updates the price of a given market. This is a good indicator of the overall health of a given market.
+* [`side_car_health_check_provider_updates_total`](#side_car_health_check_provider_updates_total): This metric is a counter that increments everytime the side-car utilizes a given providers market data. This is a good indicator of the health of a given provider. Note that providers may not be responsible for every market. However, the side-car correctly tracks the number of expected updates for each provider.
+
+### `side_car_health_check_system_updates_total`
+
+This metric should be monotonically increasing. Specifically, the rate of this metric should be inversely correlated to the configured `UpdateInterval` in the oracle side-car configuration (`oracle.json`). To check this, you can run the following query in Prometheus:
```

Comment:
instead of "should be increasing"

I think we should use "is monotonically increasing"

There is no case where it is decreasing.

Same with below